### PR TITLE
fix: do not preload & poll for NFT balances by default

### PR DESCRIPTION
**Pull request by meeh0w**
* [CP-9250](https://ava-labs.atlassian.net/browse/CP-9350)

## Description
This should help with extension getting laggy when users have a lot of NFTs (i.e. people that use liquidity pools often - they end up with thousands of LBT (Liquidity Book Token)

* Related PR: https://github.com/ava-labs/vm-modules/pull/174

## Changes
* Changes `BalancesProvider` and related backend services so that we always decide which token types to fetch balances for:
    * The `WalletBalances` item (summary at the top) asks for native & ERC-20 tokens
    * Bridge asks for balances of bridgeable tokens (native & ERC-20)
    * Fees widget asks for native balances
    * Collectibles view asks for NFT balances
* Provides `storage` to the `<module>.getBalances(...)` call - this will help with Coingecko request spam & rate limiting

## Testing
* Verify NFT balances are not loading on the initial screen
* Verify NFTs are loading fine when in Collectibles view
* Verify you can send NFTs

## Checklist for the author
- [x] I've covered new/modified business logic with Jest test cases.
- [x] I've tested the changes myself before sending it to code review and QA.

### CODE REVIEW

**Review by bferenc — COMMENTED** [review]
Nice! Ready to approve after the dependency version update 👍

**Review by bferenc — APPROVED** [review]
👏

**Review by gergelylovas — APPROVED** [review]

File: src/background/services/balances/BalancePollingService.ts [hunk 1]
```diff
@@ -28,13 +29,19 @@ export class BalancePollingService implements OnLock, OnAllExtensionClosed {
...
     account: Account,
     activeChainId: number,
-    roundRobinChainIds: number[]
+    roundRobinChainIds: number[],
+    tokenTypes: TokenType[]
   ) {
     // Stop any polling that may be occurring already
     this.stopPolling();
     // Start a new interval
     // this._preventSchedulingNextUpdate = false;
```
Comment (vvava) [inline src/background/services/balances/BalancePollingService.ts:38]: I guess it can be removed

**Review by vvava — APPROVED** [review]

File: src/background/services/balances/BalancePollingService.ts (hunk 1 above)
Comment (meeh0w) [reply src/background/services/balances/BalancePollingService.ts:38]: Good point! Will do it in a follow-up PR.

**Pull request merged**